installer/roles/image_build/templates/Dockerfile.j2

```diff
@@ -1,75 +1,56 @@
-FROM centos:7
+FROM centos:8
 
 USER root
 
-ADD ansible.repo /etc/yum.repos.d/ansible.repo
```

Comment:
Had to switch to pip because we don't have upstream EL8 RPMs for core yet.

Reply:
EPEL?

Reply:
@wenottingham what does that actually get us vs just installing from pip?

Reply:
Closer to anything a downstream might do, at least.

Reply:
It's what the dev env does at the moment

Reply:
@wenottingham Seems like you care more than I do, made the change.